application/Directory.Packages.props

```diff
@@ -26,6 +26,7 @@
     <PackageVersion Include="NUlid" Version="1.7.1" />
     <!-- PlatformPlatform dependencies - Infrastructure -->
     <PackageVersion Include="Microsoft.EntityFrameworkCore.SqlServer" Version="$(EfCoreVersion)" />
+    <PackageVersion Include="OpenTelemetry.SemanticConventions" Version="1.0.0-rc9.9" />
```

Comment:
Not sure if this should be in the `<!-- Open Telemetry -->` section? Right now, that part is copied from Aspire.